experiment/measurer.py

```diff
@@ -67,13 +68,131 @@ def exists_in_experiment_filestore(path: pathlib.Path) -> bool:
                               must_exist=False).retcode == 0
 
 
+def get_fuzzer_benchmark_key(fuzzer: str, benchmark: str):
+    """Return the key in coverage dict for a pair of fuzzer-benchmark."""
+    return fuzzer + ' ' + benchmark
+
+
+def get_trial_ids(experiment: str, fuzzer: str, benchmark: str):
+    """Get ids of all finished trials for a pair of fuzzer and benchmark."""
+    trial_ids = [
+        trial_id_tuple[0]
+        for trial_id_tuple in db_utils.query(models.Trial.id).distinct().filter(
+            models.Trial.experiment == experiment, models.Trial.fuzzer ==
+            fuzzer, models.Trial.benchmark == benchmark,
+            ~models.Trial.preempted)
+    ]
+    return trial_ids
+
+
+def get_coverage_infomation(coverage_summary_file):
+    """Read the coverage information from |coverage_summary_file|
+    and skip possible warnings in the file."""
+    with open(coverage_summary_file) as summary:
+        return json.loads(summary.readlines()[-1])
```

Comment:
What is -1 and why do we need it?

Reply:
Bloaty will have a warning info in the first line of the file. It is to skip that warning. The last line of the file is the data we want despite any possible warnings. Or should I just ignore bloaty?